cloverage/test/cloverage/test_coverage.clj

```diff
@@ -58,8 +58,8 @@
                (wrap #'track-coverage 0 '[1 "foo" bar]))))
 
 (deftest test-wrap-map
-  (is (expand= `(capture 0 {(capture 3 :a) (capture 4 ~'apple)
-                            (capture 1 :b)  (capture 2 ~'banana)})
+  (is (expand= `(capture 0 {(capture 1 :a) (capture 2 ~'apple)
+                            (capture 3 :b)  (capture 4 ~'banana)})
```

Comment:
nit: let's fix the double space in the second line since we're touching this part anyway? :P